### DISCUSSION TITLE
How to use the check parameter in wait_for_interact?

### DISCUSSION THREAD

**Discussion by TestAccout1** [kiki7000/discord.py-components#25] [Q&A]
I want the bot to wait for a specific user to interact with the buttons by checking if the user's ID is equal to the one I want, how can I implement this using the check parameter?

**Comment by kiki7000** [comment]
```py
def check(ctx):
  return ctx.user.id == "something"
await ddb.wait_for_interact('button_click', check=check)
```
Like this

**Reply by TestAccout1** [reply]
That raises an error; `AttributeError: 'dict' object has no attribute 'user'`

**Reply by kiki7000** [reply]
Update the library please

**Reply by TestAccout1** [reply]
Thank you for your efforts, it works now
